.github/issue_template.md

```diff
@@ -0,0 +1,7 @@
+Please be sure to include:
+
+* what version of Xcode you are using: `xcodebuild -version`
```

Comment:
We should generalize this for Linux as well.

Maybe ask "What operating system you are using" and then "What version of Xcode (if macOS)"?

Reply:
Ok, i'm confused, I amended the commit, but the PR won't pick it up…

Reply:
And github finally realizes it…